sdk/core/azure-core/test/ut/transport_policy_options.cpp

```diff
@@ -571,15 +581,19 @@ namespace Azure { namespace Core { namespace Test {
 
       for (auto const& target : testUrls)
       {
-        GTEST_LOG_(INFO) << "Test " << target;
-        Azure::Core::Url url(target);
-        auto request = Azure::Core::Http::Request(Azure::Core::Http::HttpMethod::Get, url);
-        std::unique_ptr<Azure::Core::Http::RawResponse> response;
-        EXPECT_NO_THROW(
-            response = pipeline.Send(request, Azure::Core::Context::ApplicationContext));
-        if (response && response->GetStatusCode() != Azure::Core::Http::HttpStatusCode::Found)
+        // Only try to access the server if we were able to contact it earlier.
```

Comment:
It is not bad what you're future-proofing it for, but yet I don't understand how it repros consistently on Linux and not on any other platform. Applying this fix may hide the issue.
Hopefully this is not a code bug either in SDK or in libcurl, but maybe Linux CI machines do block the connection?

Reply:
I have no idea - my guess is that something in the CRL validation is choking on example.com's CRL. To diagnose it I need to turn on a bunch of extra CURL tracing that will hopefully help me understand.